apps/gallery/js/ImageEditor.js

```diff
@@ -28,6 +51,37 @@ $('edit-save-button').onclick = saveEditedImage;
 editOptionButtons.forEach(function(b) { b.onclick = editOptionsHandler; });
 editAspectMoreButtons.forEach(function(b) {b.onclick = editAspectMoreHandler;});
 
+/*
+ * These arrays represent matricies for image processing effects. They
+ * are passed to the ImageProcessor.setMatrix() method to specify color
+ * transformations of an image. The property names in this object are
+ * linked to the Gallery editor UX: if they are changed here, they also
+ * must be changed in the HTML.
+ */
+var Effects = {
```

Comment:
Moving this to make sure it's defined before use in functions. (All large blocks of changes in this PR are for the same reason)